wee_slack.py

```diff
@@ -7248,9 +7236,10 @@ def handle_getPresence(response_json, eventrouter, team, channel, metadata):
         callback=handle_prefs,
     )
     EVENTROUTER.receive(s)
+
     s = SlackRequest(
         None,
-        "users.getPresence",
+        "users.list",
```

Comment:
Not sure why you're using `users.list` here, as that method doesn't return the info we need here (`manual_away`)?